.github/workflows/Benchmarks.yml

```diff
@@ -1,13 +1,21 @@
 name: Benchmarks
 on:
+  workflow_run:
+    workflows: ["CI"]
+    types:
+      - completed
   pull_request_target:
     branches: [ master ]
```

Comment:
The `pull_request_target` is triggering the benchmarks workflow immediately, which is not desired. I'm exploring alternatives, but they might prevent the PR comments style of results vs in the checks tab. How important/strong a preference is the benchmark results in comments vs in the checks tab?

Reply:
I prefer having the comments. As you say, changing this to `pull_request` has advantages, but also means we do not get the comments if the PR is from a fork. There is some discussion about this starting at https://github.com/MilesCranmer/AirspeedVelocity.jl/issues/97#issuecomment-2936248772. If we want to switch to `pull_request` instead of `pull_request_target` we would set `job-summary: 'true' `, but then the tables are a bit hidden.